nupic/embodied/agents/curious_ppo_agent.py

```diff
@@ -324,6 +328,34 @@ def update(self):
         if self.rollout.best_ext_return is not None:
             info["performance/best_ext_return"] = self.rollout.best_ext_return
 
+        if not self.debugging:
```

Comment:
Do we always want to plot all these? In the future we can think of a way of defining when launching the experiment which are the features we want to plot to wandb. Calculating/logging might have some impact to runtime, so we could selectively select only the most relevant ones in longer runs, although I don't know if that is the case here.